packages/inputs/src/inputs/file.ts

```diff
@@ -42,7 +42,10 @@ export const file: FormKitTypeDefinition = {
             fileName('$file.name'),
             $if(
               '$value.length === 1',
-              fileRemove(icon('fileRemove'), '$ui.remove.value')
+              fileRemove(
+                icon('fileRemove'),
+                '$ui.remove.value + " " + $file.name'
```

Comment:
Hi, can this text be overridden? I prefer to have just the 'remove' text...

If not: can you make this entire line translatable?